kie-api/src/main/resources/org/kie/api/old-kmodule.xsd

```diff
@@ -0,0 +1,203 @@
+<!--
+  ~ Copyright 2016 JBoss Inc
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
```

Comment:
This might be what's breaking the testing. I 'll take a look tomorrow.

2016-01-12 20:05 GMT+01:00 Mario Fusco <notifications@github.com>:

> In kie-api/src/main/resources/org/kie/api/old-kmodule.xsd
> <https://github.com/droolsjbpm/droolsjbpm-knowledge/pull/117#discussion_r49498107>
> :
>
> > +<!--
> > +  ~ Copyright 2016 JBoss Inc
> > +  ~
> > +  ~ Licensed under the Apache License, Version 2.0 (the "License");
> > +  ~ you may not use this file except in compliance with the License.
> > +  ~ You may obtain a copy of the License at
> > +  ~
> > +  ~      http://www.apache.org/licenses/LICENSE-2.0
> > +  ~
> > +  ~ Unless required by applicable law or agreed to in writing, software
> > +  ~ distributed under the License is distributed on an "AS IS" BASIS,
> > +  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +  ~ See the License for the specific language governing permissions and
> > +  ~ limitations under the License.
> > +  -->
> > +
>
> @ge0ffrey <https://github.com/ge0ffrey> It's wrong to put a comment
> before the xml header. Please remove it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/droolsjbpm/droolsjbpm-knowledge/pull/117/files#r49498107>
> .
>



-- 
With kind regards / Met vriendelijke groeten,
Geoffrey De Smet